src/execute_request.jl

```diff
@@ -58,30 +56,50 @@ const displayqueue = Any[]
 
 # remove x from the display queue
 function undisplay(x)
-    i = findfirst(equalto(x), displayqueue)
-    if i > 0
+    i = findfirst(isequal(x), displayqueue)
+    if i !== nothing && i > 0
         splice!(displayqueue, i)
     end
     return x
 end
 
+if VERSION < v"0.7.0-DEV.3498" # julia #25544
+    import Base.REPL: ip_matches_func
+else
+    import Base: ip_matches_func
+end
+
 function show_bt(io::IO, top_func::Symbol, t, set)
     # follow PR #17570 code in removing top_func from backtrace
-    eval_ind = findlast(addr->Base.REPL.ip_matches_func(addr, top_func), t)
-    eval_ind != 0 && (t = t[1:eval_ind-1])
+    eval_ind = findlast(addr->ip_matches_func(addr, top_func), t)
```

Comment:
Similar to above, use `Compat.findlast` and then you can skip the `eval_ind != 0` check.